pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -263,6 +263,11 @@ in rec {
         ln -s libsandbox.1.tbd $out/lib/libsandbox.tbd
       '';
     };
+
+    simd = stdenv.mkDerivation {
```

Comment:
```suggestion
    simd = stdenvNoCC.mkDerivation {
```

Reply:
I also marked this one to build locally since its only purpose is to provide an unversioned reference that `overrideSDK` can replace.